envoy/buffer/buffer.h

```diff
@@ -458,6 +458,13 @@ class Instance {
     writeInt<ByteOrder::BigEndian, T, Size>(value);
   }
 
+  /**
+   * Copy multiple string type fragments to the buffer.
+   * @param fragments A set of string view with variable length.
```

Comment:
nit view -> views

Reply:
get it.

Reply:
:) actually it's not a set; it's a sequence.